src/shared/services/order/order.service.spec.ts

```diff
@@ -67,6 +67,26 @@ describe('OrderService', () => {
   });
 
```

Comment:
Can we add a test where some of the fixed list field types do and some do not have order with them please? Like the example we discussed today with Aman:
`DD/01/YYYY        regionalCentreEnum    CARDIFF    Cardiff    5
DD/01/YYYY        regionalCentreEnum    MANCHESTER    Manchester    2
DD/01/YYYY        regionalCentreEnum    OXFORD    Oxford`